content/docs/tutorials/aws/aws-ts-assume-role.md

```diff
@@ -2,7 +2,7 @@
 title: "AWS AssumeRole Example"
 ---
 
-<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/aws-ts-assume-role" target="_blank">
+<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/aws-ts-assume-role/assume-role" target="_blank">
```

Comment:
I think we need a way to disable the button for certain examples. This particular example has two Pulumi programs: one in `assume-role` and the other in `create-role`, thus, we've chosen not to have the button in this example's README.md.

Reply:
Added some more details in a comment below. Perhaps we should just search inside the example's README.md to see if the button is present. Otherwise, don't show the button at all.